subscriptions/fixed-price/checkout/node/src/server.ts

```diff
@@ -27,55 +33,114 @@ const endpointSecret = process.env.DIAGONAL_WEBHOOK_ENDPOINT_SECRET as string
 
 const diagonal = new Diagonal(apiKey)
 
-// Create an account for your customer
-app.post('/create-account/', async (req, res) => {
-  const email = req.body.email
-  // const name = req.body.name
-  // ...
+/**
+ * Plans
+ * Since you will be integrating fixed price subscriptions
+ * you will need some notion of plans/products, either defined locally, in your database,
+ * or using an external provider. In this example we choose to define plans locally.
+ */
+const basicPlan = {
+  id: '1',
+  amount: '10',
+  tokens: [Token.DAI, Token.USDC],
+  chains: [Chain.ETHEREUM, Chain.POLYGON],
+  interval: 'month' as const,
+  intervalCount: 1,
+}
 
-  // step 1: create diagonal customer
-  const customer = await diagonal.customers.create({
-    email,
-  })
+const premiumPlan = {
+  id: '2',
+  amount: '50',
+  tokens: [Token.DAI, Token.USDC],
+  chains: [Chain.ETHEREUM, Chain.POLYGON],
+  interval: 'month' as const,
+  intervalCount: 1,
+}
 
-  // step 2: create a user in your database, store Diagonal customer id along side it
-  // createUser(email, name, customer.id, ...)
+/**
+ * Checkout session
+ *
+ *  IMPORTANT fields
+ *  - `customer_id`: WHO is paying
+ *                   We recommend you pass a Diagonal customer id on checkout creation.
+ *                   `customer_id` will help you link a paying subscriber to a completed checkout session.
```

Comment:
nit: 
We recommend you pass a Diagonal customer id when creating a checkout session.
 *                   `customer_id` will help you link a paying subscriber to a completed checkout session.